Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownTextBlock.cs

```diff
@@ -50,6 +51,12 @@ public sealed class MarkdownTextBlock : Control, ILinkRegister
         public event EventHandler<LinkClickedEventArgs> LinkClicked;
 
         /// <summary>
+        /// Fired when an image from the markdown document needs to be resolved.
+        /// The default implementation is basically <code>new BitmapImage(new Uri(e.Url));</code>.
+        /// </summary>
+        public event EventHandler<ResolveImageEventArgs> ResolveImage;
```

Comment:
I'm not a big fan of the name, it doesn't sound like an event. How about something along the lines of ImageRequested or ImageResolving?